backend/src/server.js

```diff
@@ -40,7 +41,17 @@ import { restrictCors } from './middlewares'
   app.use(bodyParser.json());
   app.use(restrictCors);
   app.use('/api/v1', routers.v1Router(express));
-
+  // route to handle errors
```

Comment:
Handle which error?
Does it work?
How does it work?

Reply:
so  if a user tries any route eg
{{base_url}}/resource since the path does not exist, the handler is triggered